recipes/bitpack

```diff
@@ -0,0 +1,3 @@
+(bitpack :fetcher github
+         :repo "skeeto/bitpack"
+         :files ("bitpack.el" "README.md" "UNLICENSE"))
```

Comment:
It's not necessary to include the license or README (MELPA avoids doing so by default).  I think you can use the default recipe here instead, which will automatically exclude your tests:

```suggestion
(bitpack :fetcher github :repo "skeeto/bitpack")
```